.github/workflows/test_and_coverage.yml

```diff
@@ -26,6 +26,11 @@ jobs:
         pip install -r example/requirements.txt
     - name: Run Tox
       run: tox
+    - name: Run isort and black
+      # needs to be run outside of pytest else only the example app is tested
+      run: |
+        isort .
+        black .
```

Comment:
Or via `pytest-black`?

Reply:
pytest-black is also enabled, but only tests the `example` folder... :-/